src/test/js/reporters/accessibility-reporter/Report.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+<head>
+  <meta charset="utf-8">
+  <!-- <meta http-equiv="Content-Security-Policy"
+      content="default-src *; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval'
+       https://cdnjs.cloudflare.com https://ajax.googleapis.com http://www.google.com"> -->
+  <script src="resources/angular.min.js"></script>
+
+  <title>EXUI Web Accesibility Test Report</title>
```

Comment:
Can we change to IDAM from EXUI?

Reply:
This has been updated in the latest push